fabric/src/main/resources/assets/lambdacontrols/lang/zh_cn.json

```diff
@@ -0,0 +1,130 @@
+{
+  "key.lambdacontrols.look_down": "视角下移",
+  "key.lambdacontrols.look_left": "视角左移",
+  "key.lambdacontrols.look_right": "视角右移",
+  "key.lambdacontrols.look_up": "视角上移",
+  "lambdacontrols.action.attack": "攻击",
+  "lambdacontrols.action.back": "向后移动",
+  "lambdacontrols.action.chat": "打开聊天栏",
+  "lambdacontrols.action.drop_item": "丢弃所选物品",
+  "lambdacontrols.action.exit": "退出",
+  "lambdacontrols.action.forward": "向前移动",
+  "lambdacontrols.action.hit": "挖掘",
+  "lambdacontrols.action.hotbar_left": "向左循环选择快捷栏",
+  "lambdacontrols.action.hotbar_right": "向右循环选择快捷栏",
+  "lambdacontrols.action.inventory": "物品栏",
+  "lambdacontrols.action.jump": "跳跃",
+  "lambdacontrols.action.left": "向左移动",
+  "lambdacontrols.action.pause_game": "暂停游戏",
+  "lambdacontrols.action.pick_block": "选取方块",
+  "lambdacontrols.action.pickup": "拿取一个/拿取一半",
+  "lambdacontrols.action.pickup_all": "拿取一组/拿取全部",
+  "lambdacontrols.action.place": "放置方块",
+  "lambdacontrols.action.player_list": "玩家列表",
+  "lambdacontrols.action.quick_move": "快速移动物品",
+  "lambdacontrols.action.right": "向右移动",
+  "lambdacontrols.action.screenshot": "截图",
+  "lambdacontrols.action.sneak": "潜行",
+  "lambdacontrols.action.sprint": "疾跑",
+  "lambdacontrols.action.swap_hands": "与副手交换",
+  "lambdacontrols.action.toggle_perspective": "切换视角",
+  "lambdacontrols.action.toggle_smooth_camera": "切换电影视角",
+  "lambdacontrols.action.use": "使用物品/放置方块",
+  "lambdacontrols.action.zoom": "视野缩放",
+  "lambdacontrols.action.zoom.in": "缩放时将视野推近",
+  "lambdacontrols.action.zoom.out": "缩放时将视野拉远",
+  "lambdacontrols.action.zoom.reset": "缩放时重置缩放距离",
+  "lambdacontrols.button.a": "A",
+  "lambdacontrols.button.b": "B",
+  "lambdacontrols.button.x": "X",
+  "lambdacontrols.button.y": "Y",
+  "lambdacontrols.button.left_bumper": "左肩键",
+  "lambdacontrols.button.right_bumper": "右肩键",
+  "lambdacontrols.button.back": "选择键",
+  "lambdacontrols.button.start": "开始键",
+  "lambdacontrols.button.guide": "功能键",
+  "lambdacontrols.button.left_thumb": "左摇杆（按压）",
+  "lambdacontrols.button.right_thumb": "右摇杆（按压）",
+  "lambdacontrols.button.dpad_up": "十字键上",
+  "lambdacontrols.button.dpad_right": "十字键右",
+  "lambdacontrols.button.dpad_down": "十字键下",
+  "lambdacontrols.button.dpad_left": "十字键左",
+  "lambdacontrols.axis.left_x+": "左摇杆右（X轴正向）",
+  "lambdacontrols.axis.left_y+": "左摇杆上（Y轴正向）",
+  "lambdacontrols.axis.right_x+": "右摇杆右（X轴正向）",
+  "lambdacontrols.axis.right_y+": "右摇杆上（Y轴正向）",
+  "lambdacontrols.axis.left_trigger": "左扳机键",
+  "lambdacontrols.axis.right_trigger": "右扳机键",
+  "lambdacontrols.axis.left_x-": "左摇杆左（X轴负向）",
+  "lambdacontrols.axis.left_y-": "左摇杆下（Y轴负向）",
+  "lambdacontrols.axis.right_x-": "右摇杆左（X轴负向）",
+  "lambdacontrols.axis.right_y-": "右摇杆下（Y轴负向）",
+  "lambdacontrols.button.unknown": "未知（%d）",
+  "lambdacontrols.controller.connected": "手柄%d已连接。",
```

Comment:
Isn't there some missing spacing around the `%d`?

Reply:
Note: I'm just clueless, I just learned that in this language spacing isn't required :)